torch/ao/quantization/fx/prepare.py

```diff
@@ -696,13 +696,6 @@ def maybe_insert_observers_before_graph_output(
     for those nodes.
     """
 
-    # TODO(future PR): update the output_quantized_idxs API to match
```

Comment:
can we leave this comment in the code, since this PR is not adding support for arbitrary data structures?

Reply:
Sure